awesome-books-with-ES6/modules/addBook.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable import/no-cycle */
+/* eslint-disable max-len */
+import Book from './Book.js';
```

Comment:
- [ ] Kindly enable linters here
- [ ] For the `max-len` error simply break up the length of your comment in line 12 into multi-line comments.